### ISSUE TITLE
Help in the development of a hero and win 100$

### ISSUE THREAD

**Issue by rafaelcastrocouto** [rafaelcastrocouto/dotacard#9]
Update: Help in the development of a hero and win 100$

Github Issue https://github.com/rafaelcastrocouto/dotacard/issues/9
Bountysouce [$100 reward](https://www.bountysource.com/issues/6711442-help-in-the-development-of-a-hero-and-win-100)
## More info on our [wiki page: How to develop a hero](https://github.com/rafaelcastrocouto/dotacard/wiki/Dotacard-Wiki#how-to-develop-a-new-hero)

Original issue:

Add some FX like this http://jackrugile.com/pongoo
In other words, create a canvas on top of the map and create methods for particle effects. 
Current abilities that should have FX: Rot, Crystal Nova, Frostbite, Freezing Field, Illuminate, Mana Leak, Chakra Magic, Blinding Light, Entangling Claws, Wraithfire Blast, Reincarnation, Blink and Mana Void.
If someone feels like doing this, plz make a new issue (there will be rewards as well!)

**Comment by rafaelcastrocouto** [comment]
I added particle effects, and since I can't change the issue on bountysource I'm changing this one.
From now on, the $100 will go to anyone that helps with the creation of a new hero.

**Comment by timkrief** [comment]
Could you explain how we could create a new hero ?

**Comment by rafaelcastrocouto** [comment]
I just updated the wiki with a ["How to develop a new hero"](https://github.com/rafaelcastrocouto/dotacard/wiki/How-to-develop-a-new-hero) section. Hope it helps!
If you need any extra help you can open an issue on a specific hero or just mail me! (rafaelcastrocouto at gmail dot com)

**Comment by rafaelcastrocouto** [comment]
Thx @xinton! venge in online
